boa/vyper/contract.py

```diff
@@ -623,6 +623,8 @@ def vyper_stack_trace(self, computation):
 
         child = computation.children[-1]
         child_obj = self.env.lookup_contract(child.msg.code_address)
+        if not child_obj:
+            return StackTrace(ret)
```

Comment:
interesting -- i was thinking about this and instead of truncating, do we want to try to recurse and fill in the missing stack frames with something like "unknown location in unknown contract 0x00..ab"? i'm ok either way but the latter seems like better UX. this could be important if the external contract has called back into known contracts.